packages/color-field/README.md

```diff
@@ -0,0 +1,110 @@
+## Description
+
+`sp-color-field` components are text boxes that allow users to input custom color values. Various decorations can be displayed around the color-field. It also has an attribute named `viewColor`, which allows users to see the color that has been entered into the field.
+
+### Usage
+
+[![See it on NPM!](https://img.shields.io/npm/v/@spectrum-web-components/color-field?style=for-the-badge)](https://www.npmjs.com/package/@spectrum-web-components/color-field)
+[![How big is this package in your project?](https://img.shields.io/bundlephobia/minzip/@spectrum-web-components/color-field?style=for-the-badge)](https://bundlephobia.com/result?p=@spectrum-web-components/color-field)
+
+```
+yarn add @spectrum-web-components/color-field
+```
+
+Import the side effectful registration of `<sp-color-field>` via:
+
+```
+import '@spectrum-web-components/color-field/sp-color-field.js';
+```
+
+When looking to leverage the `ColorField` base class as a type and/or for extension purposes, do so via:
+
+```
+import { ColorField } from '@spectrum-web-components/color-field';
+```
+
+## Example
+
+### Default
+
+```html
+<sp-color-field></sp-color-field>
+```
+
+### viewColor
+
+When devMode is true, the color handle will be rendered. This is useful for development and debugging purposes.
+
+```html
+<sp-color-field viewColor="true"></sp-color-field>
+```
+
+### Read Only
+
+A readonly color field
+
+```html
+<sp-color-field readonly value="#ffff00"></sp-color-field>
+```
+
+### Quiet
+
+A Quiet color field
+
+```html
+<sp-color-field quiet value="#ffff00"></sp-color-field>
+```
+
```

Comment:
Not sure if there should be more of a top level `## Validation` section, or whether the following two sections belong as `##` instead of `###` on their own...